### PR TITLE
feat(tab): Improved mixins

### PR DESCRIPTION
**Pull request by patrickrodee**

### CODE REVIEW

File: packages/mdc-tab/mdc-tab.scss [hunk 1]
```diff
@@ -129,13 +127,12 @@
 }
 
 .mdc-tab--active {
-  @include mdc-tab-text-label-color(primary);
```
Comment (moog16) [inline packages/mdc-tab/mdc-tab.scss]: This breaks CSS Vars for label and icon color. We might need to revert this change.

Comment (patrickrodee) [reply packages/mdc-tab/mdc-tab.scss]: There's a PR to fix this I think: https://github.com/material-components/material-components-web/pull/4688
